derp/derp_server.go

```diff
@@ -120,7 +122,7 @@ type Server struct {
 	multiForwarderCreated    expvar.Int
 	multiForwarderDeleted    expvar.Int
 	removePktForwardOther    expvar.Int
-	averageQueueDuration     expvar.Float
+	avgQueueDuration         *uint64 // In milliseconds
```

Comment:
I'd just inline the uint64 here. (drop the *, and use & where you needs its address)

Reply:
but then because it's atomic, the convention is use mention so in a comment:

```
    // in milliseconds; accessed atomically
```